src/main/java/models/Player.java

```diff
@@ -65,5 +66,9 @@ public void winBet(){
         money = money + bet;
     }
 
+    public void doubleDown(int setBet)
+    {
+        money = (setBet * 2);
+    }
```

Comment:
Double down doesn't need to take a int. All it should is double bet and store it back into bet.